### PR TITLE
Feature/29970 school staff unavailable

### PR DESCRIPTION
**Pull request by danwilliams**

### CODE REVIEW

File: admin/availability/middleware.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return next()
+  }
+  const currentDate = moment.utc()
+  const checkWindow = await checkWindowV2Service.getActiveCheckWindow()
+  if (!checkWindow || !currentDate.isBetween(checkWindow.adminStartDate, checkWindow.adminEndDate)) {
+    res.locals.pageTitle = 'Unavailable'
+    return res.render('availability/admin-window-unavailable', {
+      currentDate,
+      checkWindow,
+      startDateFormatted: checkWindow.adminStartDate.format('dddd D MMMM YYYY')
```
Comment (GeorgeChatzigiannis) [inline admin/availability/middleware.js]: What is this date format?
Is this a GDS approved one?

Comment (jon-shipley) [reply admin/availability/middleware.js]: Please use the date service to format dates, preferably using an existing known format.

File: admin/views/availability/admin-window-unavailable.ejs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+<div class="grid-row" aria-describedby="lead-paragraph">
+    <div class="column-two-thirds">
+
+        <%- partial('partials/_session_expiry') %>
+
+        <header class="page-header">
+            <h1 class="heading-xlarge">
+                <% if(checkWindow && currentDate.isBefore(checkWindow.adminStartDate)) { %>
+                The service is unavailable
```
Comment (jon-shipley) [inline admin/views/availability/admin-window-unavailable.ejs]: I'm not sure this check is giving us much...

Comment (GeorgeChatzigiannis) [reply admin/views/availability/admin-window-unavailable.ejs]: In addition whichever behavior is picked up for this shouldn't it be part of a presenter?

Comment (danwilliams) [reply admin/views/availability/admin-window-unavailable.ejs]: The prototypes specify two difference messages hence this code.

File: admin/availability/middleware.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+'use strict'
```
Comment (GeorgeChatzigiannis) [inline admin/availability/middleware.js]: What about adding a test spec for the middleware?

**Pull request merged**